python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -237,6 +237,8 @@ def _getDefaultValueInternal(self, attr, val=None, check=False):
         from Ganga.Utility.Config import Config
         is_finalized = Config._after_bootstrap
 
+        _useDefVal = False
```

Comment:
I was playing with objects, this is what was left, I can re-name if necessary but yes it's never exposed or changed externally